_posts/2018-09-19-week02.md

```diff
@@ -8,10 +8,10 @@ title: Week 2
 ## Thoughts
 
 ### Brief Summary
-> The article was about a dispute that arose pertaining to if Catalina Cruz (politican) should recieve a Wikipedia page before winning the Democratic Primary. The disagreement between editors lead to the permanent ban of an editor under the name "BrillLyle".
+> The article was about a dispute that arose pertaining to whether Catalina Cruz (politican) should recieve a Wikipedia page before winning the Democratic Primary. The disagreement between editors led to the permanent ban of an editor under the name "BrillLyle".
 
```

Comment:
@f0cus10 
@Jimmyzs 
 was wondering whether someone would catch the "if" misuse. Good catch.